docs/monthly/2024-01-monthly.md

```diff
@@ -12,150 +12,97 @@ developments to gather their feedback on proposed plans.
-  it's content.
+* Renamed 'Drop Babbage support #1178' to [Switch layer 2 ledger to Conway #1178](https://github.com/input-output-hk/hydra/issues/1178) to better capture its content.
 
 ## Hydra development
 
-[Issues and pull requests closed since the last
-report](https://github.com/input-output-hk/hydra/issues?q=is%3Aclosed+sort%3Aupdated-desc+closed%3A2023-11-30..2024-01-31)
+[Issues and pull requests closed since the last report](https://github.com/input-output-hk/hydra/issues?q=is%3Aclosed+sort%3Aupdated-desc+closed%3A2023-11-30..2024-01-31).
 
 This month, the team worked on the following:
 
 ### API changes: dropping JSON Tx
 
-The Hydra API now uses a TextEnvelope containing the upstream CBOR encoding
-from cardano-api for the `Tx` type and has removed the JSON representation from
-the API [#1240](https://github.com/input-output-hk/hydra/pull/1240). This is so
-transaction types have a fixed canonical encoding which will not change between
-versions. Consequently, consumers of the Hydra API will need to deserialise
-CBOR in order to inspect transactions.
+The Hydra API now utilizes a TextEnvelope containing the upstream CBOR encoding from `cardano-api` for the `Tx` type and removes the JSON representation from the API [#1240](https://github.com/input-output-hk/hydra/pull/1240). This ensures that transaction types have a fixed canonical encoding that will not change between versions. As a result, consumers of the Hydra API will need to deserialize CBOR to inspect transactions.
 
-This representation is equivalent to the TextEnvelope produced by cardano-cli,
-except with an additional `txId` field, that is guaranteed to match the transaction
-id within the cbor object.
+This representation mirrors the TextEnvelope produced by the Cardano CLI, with the addition of an `txId` field, which is guaranteed to match the transaction ID within the CBOR object.
 
-:::warning Breaking Change
+:::warning Breaking change
 
-We do not plan to go through a deprecation cycle on this breaking change to the API. If you
-are affected and see a problem updating your Hydra client to use CBOR, please
-[open an issue](https://github.com/input-output-hk/hydra/issues/new) on Github.
+We do not plan to go through a deprecation cycle on this breaking change to the API. If you are affected and see a problem updating your Hydra client to use CBOR, please [open an issue](https://github.com/input-output-hk/hydra/issues/new) on GitHub.
 
 :::
 
 ### Hydra Chess
-We started building a game on Hydra as dogfooding allows us to find things which
-are suboptimal or stumble accross a bug that needs fixing. Hydra Chess proved to
-be no different and we learned in the process of making this dApp.
+
+The team began developing a game on Hydra, leveraging dogfooding to uncover suboptimal elements or stumble upon bugs requiring resolution. Hydra Chess exemplified this process, offering valuable lessons during DApp development.
```

Comment:
Suggestions: allow us to use the "we" form as we used this in the past

@olgahryniuk @oduameh The rewritten text here reads fine, but it can't shake off feeling weird when writing it (while it's very natural to just say "we")